dist/angular-validation.js

```diff
@@ -665,6 +665,19 @@
         validMethod = (angular.isUndefined(validMethod)) ? $validationProvider.getValidMethod() : validMethod;
 
         /**
+         * A simple hack to provide global validate
+         */
+        if(!$validationProvider.validators){
+          $validationProvider.validators = [];
+          $validationProvider.validAll = function () {
+            return $validationProvider.validators.map(x=>x()).reduce((x,y)=>x&&y);
+          };
+        }
+        $validationProvider.validators.push(function () {
+          return checkValidation(scope, element, attrs, ctrl, validation, ctrl.$viewValue)
+        });
+
+        /**
```

Comment:
I know where I was wrong, the input needs a name to reg into form but i missed it.
So i add nameAttr to every input. After that, the '$validation.validate($scope.myForm)' finally worked.
I'll close the PR.